vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -124,6 +124,9 @@ class ViewShadowNode : public ShadowNodeBase {
   }
   void IsFocusable(bool isFocusable) {
     m_isFocusable = isFocusable;
+
+    if (IsControl())
```

Comment:
nit: Should we have curlies for legibility, or does the clang formatter remove them in case of single statement?

Reply:
Personal preference is yes but we haven't established many style guidelines for our C++ other than "follow clang-format and we like CppCoreGuidelines". Would love for us to figure some more out tbh.